nipype/pipeline/plugins/multiproc.py

```diff
@@ -137,8 +137,11 @@ def _clear_task(self, taskid):
 
     def _submit_job(self, node, updatehash=False):
         self._taskid += 1
-        if getattr(node.inputs, 'terminal_output', '') == 'stream':
-            node.inputs.terminal_output = 'allatonce'
+
+        # Don't allow streaming outputs
+        if hasattr(node.interface, 'terminal_output') and \
+                node.interface.terminal_output == 'stream':
+            node.interface.terminal_output = 'allatonce'
```

Comment:
This might be something I messed up with my merge, but I think the `getattr` approach is cleaner.

```Python
# Don't allow streaming outputs
if getattr(node.interface, 'terminal_output', '') == 'stream':
    node.interface.terminal_output = 'allatonce'
```